src/pages/smart-collection/store/sagas.ts

```diff
@@ -92,10 +89,10 @@ function* fetchUnderTheRadar() {
 }
 
 function* fetchMostLoved() {
-  const tracks = yield call(Explore.getTopFolloweeSaves)
+  const tracks = yield* call(Explore.getTopFolloweeSaves)
 
   const trackIds = tracks
```

Comment:
can probably share this logic

Reply:
Probably, though I'm trying to keep the functional change of this PR minimal and keep it mostly to typing